qm-delsnapshot: add page (#8852)
tldr-pages/tldr: pages/linux/qm-delsnapshot.md

# qm delsnapshot

> > Delete a virtual machine snapshot.

Reviewer: ```suggestion
> Delete virtual machine snapshots.
```

> More information: <https://pve.proxmox.com/pve-docs/qm.1.html>.

- Delete a snapshot:

`qm delsnapshot {{vm_id}} {{snapshot_name}}`

> - Delete a snapshot from config even if removing disk snapshot fails:

Reviewer 2: ```suggestion
- Delete a snapshot from config (even if removing the disk snapshot fails):
```
I think this would be better. What do you think?

> - Delete a snapshot from config (even if removing the disk snapshot fails):

Reviewer: Perhaps this better because we don't abbreviations in example descriptions
```suggestion
- Delete a snapshot from a configuration file (even if removing the disk snapshot fails):
```
